components/kyma-environment-broker/internal/hyperscaler/azure/client.go

```diff
@@ -42,11 +65,54 @@ func (nc *AzureClient) CreateNamespace(ctx context.Context, azureCfg *Config, gr
 	// we need to use a copy of the location, because the following azure call will modify it
 	locationCopy := azureCfg.GetLocation()
 	parameters := eventhub.EHNamespace{Location: &locationCopy, Tags: tags}
-	ehNamespace, err := nc.createAndWait(ctx, groupName, namespace, parameters)
+	ehNamespace, err := nc.createAndWaitNamespace(ctx, groupName, namespace, parameters)
```

Comment:
```suggestion
	ehNamespace, err := nc.createNamespaceAndWait(ctx, groupName, namespace, parameters)
```

Reply:
Why, is it a convention/pattern?

Reply:
The naming convention used in the `client.go` file follows this pattern `VerbSubject` like `AzureInterface.CreateNamespace` or `AzureInterface.CreateResourceGroup`. Should we follow the same pattern for consistency?

Reply:
OK, done